src/appComponents/tracks/TracksController.js

```diff
@@ -52,16 +53,17 @@ tracksModule.controller("TracksController", ["$mdDialog","$sessionStorage", "$ro
 
         }
 
-        tsc.showSession = function(track, event) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2431821597/issues/source?bid=3233857&fileBranchId=3238420#l56)